### DISCUSSION TITLE
HTTP2 Link Headers

### DISCUSSION THREAD

**Discussion by samuel-a-cooper** [vercel/next.js#11892] [Help]
Hey,

When using a reverse proxy (Nginx in my instance) in front of Next, would using HTTP2 pushing to get resources to the client quicker be appropriate?

If so, due to resources paths changing with differing builds, would it be possible to add Link preload headers to a pages response, without the use of a custom server, so that directives like Nginxs `http2_push_preload` can be used.

Thanks for your time!

**Comment by iFwu** [comment]
Also interested in HTTP2 server push with Next.js. https://github.com/dunglas/vulcain is a good case to study.
